### PR TITLE
Mm 40894 not able to download license from download history

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
This PR removes the value _new from anchor target property. _new is not a valid value, it just works to open a new tab window because it tries to open an existing tab named _new, since it fails to find it, it results in opening a new empty tab. Also, replacing it to _self so it works fine for downloading invoices from the desktop app which is the place where originally the problem was found.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-40894

#### Related Pull Requests
https://github.com/mattermost/mattermost-webapp/pull/9645

#### Screenshots

https://user-images.githubusercontent.com/10082627/152431906-80358f2c-9f7e-4fd5-927d-f4a274fa035e.mp4

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/admin_console/billing/billing_history.tsx [hunk 1]
```diff
@@ -151,75 +147,77 @@ const BillingHistory: React.FC<Props> = () => {
...
+                                    style='currency'
+                                    currency='USD'
+                                />
+                            </td>
+                            <td>
+                                {getPaymentStatus(invoice.status)}
+                            </td>
+                            <td className='BillingHistory__table-invoice'>
+                                <a
+                                    target='_self'
```
Comment (pablovelezvidal) [inline components/admin_console/billing/billing_history.tsx:211]: This is the real change of this PR, setting back the target value to _self so it downloads the invoices in the desktop app

File: components/admin_console/billing/billing_history.test.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                <BillingHistory/>
+            </Provider>,
+        );
+
+        const invoiceTableRows = wrapper.find('table.BillingHistory__table tr.BillingHistory__table-row');
+
+        expect(invoiceTableRows.length).toBe(2);
+    });
+
+    test('Billing history section download button has the target property set as _self so it works well in desktop app', () => {
```
Comment (pablovelezvidal) [inline components/admin_console/billing/billing_history.test.tsx:132]: Added unit tests to cover this scenario from a future regression by again changing the target property value

**Review by stevemudie — APPROVED** [review]
It works!

Looks good @pablovelezvidal! I was able to download the invoice from both the Subscription and the Billing Information sections of the System Console using the Desktop app without any issues.

**Review by devinbinnie — APPROVED** [review]

**Review by neallred — APPROVED** [review]
lgtm!

**Pull request merged**